HotelManagerment/com.spring.sample/src/com/spring/sample/configuration/SecurityConfig.java

```diff
@@ -94,9 +96,10 @@ protected void configure(HttpSecurity http) throws Exception {
 				.and().exceptionHandling()
 				.accessDeniedHandler(accessDeniedHandler)
 				.and().csrf()
-				.and().headers()
+				.and().headers();
+				/*
```

Comment:
Đã chia tay thì trả quà đi em, comment làm j nữa, xóa mạnh tay vào

Reply:
An toàn là trên hết, em sợ bug mấy cái security này lắm rồi